fixtures/user.js

```diff
@@ -0,0 +1,58 @@
+const sqlFixtures = require('sql-fixtures');
+const cleandb = require('./cleandb');
+
+const dbConfig = {
```

Comment:
You will need this config a more files. Please create a file in the `test` folder responsible for loading a fixture file. Put there DB connection code as well as the function at line 55 in this file.